example/views/index.jsx

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
You need this: https://github.com/complate/complate-jsx/blob/da474a45c2877fd365fb5593b957105bf80fcce1/samples/.eslintrc#L8

🤔 Perhaps we should package the parts of that ESLint config so users just need to reference that:
```
extends:
- fnd # optional; might as well be semistandard
- plugin:react/recommended
```